packages/manager/apps/sign-up/src/routing.js

```diff
@@ -61,6 +62,41 @@ export const state = {
       ssoAuthentication.logout($location.search().onsuccess);
     },
 
+    isKycFeatureAvailable: /* @ngInject */ ($http) => {
+      return $http
+        .get(`/feature/identity-documents/availability`, {
+          serviceType: 'aapi',
+        })
+        .then(
+          ({ data: featureAvailability }) =>
+            featureAvailability['identity-documents'],
+        );
+    },
+
+    kycStatus: /* @ngInject */ ($http, isKycFeatureAvailable) => {
+      if (isKycFeatureAvailable) {
+        return $http.get(`/me/procedure/identity`).then(({ data }) => data);
+      }
+      return false;
+    },
+
+    needkyc: /* @ngInject */ (isKycFeatureAvailable, kycStatus) => {
+      if (isKycFeatureAvailable) {
+        return ['required', 'open'].includes(kycStatus.status);
+      }
+      return false;
+    },
+
+    goToKycDocumentUploadPage: /* @ngInject */ (
+      $window,
+      coreURLBuilder,
+    ) => () => {
+      set(
```

Comment:
Avoid adding lodash dependencies, this can be done easily by assign the new value to $window.location.href

Reply:
Done Jack